dashboard/app/controllers/api_controller.rb

```diff
@@ -256,6 +257,15 @@ def section_assessments
 
   private
 
+  ONE_HOUR = 3600
+
+  def configure_caching
+    return unless Script.should_cache?
+    max_age = DCDO.get('pegasus_document_max_age', ONE_HOUR * 4)
+    proxy_max_age = DCDO.get('pegasus_document_proxy_max_age', ONE_HOUR * 2)
+    response.headers['Cache-Control'] = "public,max-age=#{max_age},s-maxage=#{proxy_max_age}"
```

Comment:
Can we add a versioning key to this response and cache it forever?  This would also give us immediate expiration when the data changes.